src/main/java/billgates/view/gui/AddEntryDialog.java

```diff
@@ -0,0 +1,97 @@
+package billgates.view.gui;
+
+import com.github.lgooddatepicker.components.DateTimePicker;
+
+import javax.swing.*;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+public class AddEntryDialog extends JDialog {
+
+    private int result;
+    private ZonedDateTime date;
+    private static double value;
+    private String currency;
+    private String description;
+    private String from;
+    private String to;
+    private String location;
+    private final MainFrame mainFrame;
+
+    public AddEntryDialog(MainFrame owner) {
+        super(owner);
+        this.mainFrame = owner;
+        // Create a datetime picker.
+        DateTimePicker dateTimePicker1 = new DateTimePicker();
+        add(dateTimePicker1);
+
+        JTextField valueField = new JTextField(20);
+        JTextField currencyField = new JTextField(20);
+        JTextField descriptionField = new JTextField(20);
+        JTextField fromField = new JTextField(20);
+        JTextField toField = new JTextField(20);
+        JTextField locationField = new JTextField(20);
+
+        JPanel myPanel = new JPanel();
+        myPanel.setLayout(new BoxLayout(myPanel, BoxLayout.PAGE_AXIS));
+        myPanel.add(new JLabel("date and time:"));
+        myPanel.add(dateTimePicker1);
+        myPanel.add(new JLabel("value:"));
+        myPanel.add(valueField);
+        myPanel.add(new JLabel("currency:"));
+        myPanel.add(currencyField);
+        myPanel.add(new JLabel("description:"));
+        myPanel.add(descriptionField);
+        myPanel.add(new JLabel("from:"));
+        myPanel.add(fromField);
+        myPanel.add(new JLabel("to:"));
+        myPanel.add(toField);
+        myPanel.add(new JLabel("location:"));
+        myPanel.add(locationField);
+
+        result = JOptionPane.showConfirmDialog(null, myPanel,
+                "Information about the new entry", JOptionPane.OK_CANCEL_OPTION);
```

Comment:
```suggestion
                "Information about the new entry", JOptionPane.OK_CANCEL_OPTION, JOptionPane.PLAIN_MESSAGE);
```

Declaring that it is a `PLAIN_MESSAGE` helps remove the green question mark

Reply:
<img width="285" alt="image" src="https://user-images.githubusercontent.com/8016180/205520102-d9984d48-80a2-4c37-946d-471956cbacf8.png">

Now it looks like this, although I still didn't get rid of the center aligned text issue. 

Also, can we add a "time" label on top of the time selector?